workflows/plugins/output_loaders/entities/consensus_genome_output.py

```diff
@@ -32,7 +32,7 @@ async def load(
         workflow_outputs: dict[str, JSONValue],
     ) -> None:
         sars_cov_2 = raw_inputs.get("sars_cov_2") or raw_inputs.get("creation_source") == "SARS-CoV-2 Upload"
-        wgs = raw_inputs.get("creation_source") == "Viral CG Upload"
```

Comment:
I'm curious why this check doesn't work?

Reply:
I'm not sure. I'm going to print this out when I'm testing in sandbox.